index.html

```diff
@@ -3286,9 +3302,18 @@ <h5>Existing implementations</h5>
 <section id="capability-wrap-pan" data-ucr-role="capability">
 <h4>Wrap/duplicate data tiles when panning around the globe</h4>
 <p>
-
+A web map is not subject to the same limitations of printed maps
+and can be panned infinitely along the horizontal axis.
+Tiles should be wrapped along the X axis
```

Comment:
Maybe re-word to avoid talking specifically about the X axis. That's how it works in the standard web Mercator views, but conceptually it could wrap around the poles or in other ways for other projections.